README.md

```diff
@@ -94,6 +94,10 @@ Encode the transaction object for multi-signing.
 }
 ```
 
+## Browser Compatibility
+
+`ripple-binary-codec` is compatible with Browsers that support `BigInt` types.
```

Comment:
I think we should say that all new browsers support BigInt, but Internet Explorer and old Safari (versions prior to Safari 14) do not. I got this info from caniuse.com